pkgs/top-level/all-packages.nix

```diff
@@ -5588,6 +5588,10 @@ let
 
   node_webkit = node_webkit_0_9;
 
+  nwjs_0_12 = callPackage ../development/tools/node-webkit/nw12.nix {
```

Comment:
Hmm, how do you have this in mind? Make a derivation?
Do note that nwjs has a different install phase than the others.